WMIAdventure/frontend/src/js/components/battle/atoms/BuffPreviewBlock/styled-components/Icon.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import powerBuff from "../../../../../../assets/icons/power-buff.svg";
+import damageBuff from "../../../../../../assets/icons/damage-buff.svg";
+import shieldBuff from "../../../../../../assets/icons/shield-buff.svg";
+import healBuff from "../../../../../../assets/icons/heal-buff.svg";
+import doubleUseBuff from '../../../../../../assets/icons/double-use-buff.svg';
+
+function iconHandle(type) {
+    switch (type) {
+        case null:
+            return powerBuff;
+        case 1:
+            return damageBuff;
+        case 2:
+            return shieldBuff;
+        case 5:
+            return doubleUseBuff;
+        case 6:
+            return healBuff;
+        default:
+            return false;

```

Comment:
Tutaj tak samo, jakby były zmienne to nie trzeba byłoby potwarzać tych numerków. Domyślam się, że chodzi o ID efektów, to są zmienne w `EffectHelper.js`

Reply:
To są jednak inne identyfikatory niż te od efektów, więc możesz stworzyć nowe. Ja bym zrobił plik `BuffIds.js` w Battle do tego, bo to jest jednak powiązane z backendem i warto to zaznaczyć poza komponentem

Reply:
@emkarcinos no wiem, dlatego byłem zmieszany, te id to są id effektów, ale nie buffów, tylko efektów jakie te buffy wzmacniają